### PR TITLE
Form design

### PR DESCRIPTION
**Pull request by harshitadatra**
This PR contains 
1. Added close symbol in OOO form.After clicking on the close icon the form will close
Attaching 
image (1)
image

### CODE REVIEW

File: src/components/OOO/OOOForm.tsx
Comment (sahsisunny) [inline src/components/OOO/OOOForm.tsx:1]: Can you remove the commented code?

**Comment by sahsisunny** [comment]
Same changes(@shreya-mishra changes) in another PR(#228) why?

File: src/components/OOO/OOOForm.tsx
Comment (shreya-mishra) [reply src/components/OOO/OOOForm.tsx:1]: They has to be raise a PR from my PR because am working on different task on the same

**Comment by shreya-mishra** [comment]
> Same changes(@shreya-mishra changes) in another PR(#228) why?



They has to raise a PR from my PR because am working on different task on the same

**Review by shreya-mishra — APPROVED** [review]

File: src/components/OOO/OOOForm.tsx
Comment (shreya-mishra) [reply src/components/OOO/OOOForm.tsx:1]: removed

**Review by shreya-mishra — APPROVED** [review]

**Pull request merged**